packages/shared_preferences/shared_preferences_android/example/integration_test/shared_preferences_test.dart

```diff
@@ -493,6 +492,37 @@ void main() {
         expect(values[key], null);
       }
     });
+
+    testWidgets(
+        'old list encoding with getPreferences can be re-added with new encoding without data loss',
```

Comment:
https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#avoid-newold-modifiers-in-code

 Consider instead "Non Json list encoding"